src/Framework/QueryBuilder/Concerns/CRUD.php

```diff
@@ -22,6 +23,12 @@ trait CRUD
      */
     public function insert($data, $format = null)
     {
+        if (array_is_list($data)) {
```

Comment:
@alaca Do we have a polyfill for this somewhere? This is a PHP 8.1 function.

Reply:
@JasonTheAdams Yeah. I installed and am using the stellarwp/arrays package now.